spec/features/delete_draft_work_spec.rb

```diff
@@ -12,13 +12,22 @@
   end
 
   context 'when draft' do
-    it 'allow users to delete the work and destroys the model' do
+    it 'allow users to delete the work and destroys the model from the dashboard' do
       expect(work).to be_first_draft
       visit dashboard_path
       accept_confirm do
         click_link "Delete #{work.title}"
       end
       expect(Work.find_by(id: work.id)).to be(nil)
     end
+
+    it 'allow users to delete the work and destroys the model from the work edit page' do
+      expect(work).to be_first_draft
```

Comment:
I don't think this is a useful assertion.  We told it to be a first_draft when we created it.

Reply:
Will remove